tests/unit/accounts/test_forms.py

```diff
@@ -203,6 +203,7 @@ def test_validate_password_notok(self, db_session):
             pretend.call(
                 tag=EventTag.Account.LoginFailure,
                 ip_address=request.remote_addr,
```

Comment:
Did you also want to expand scope to encompass these kinds of changes?
```suggestion
                ip_address=request.ip_address.ip_address,
```

This particular test case is providing a local stub request, so that would have to change as well.

A thought I've been having throughout these refactors is that we're pretty-heavily stubbing a lot of things that we likely should be using fixtures for.

Reply:
next PR will probably be to remove ip_address from the call signature, so they'll go away soon. very soon.